cstar/case.py

```diff
@@ -533,16 +533,22 @@ def pre_run(self) -> None:
     def run(
         self,
         account_key=None,
-        walltime=cstar_system.environment.max_walltime,
-        queue=cstar_system.environment.primary_queue,
-        job_name="my_case_run",
-    ) -> None:
+        walltime=None,
+        queue=None,
+        job_name=None,
+    ) -> Optional["SchedulerJob"]:
         """Run the case by calling `component.run(caseroot)` on the primary component
         (to which others are coupled)."""
 
         # Assuming for now that ROMS presence implies it is the master program
         # TODO add more advanced logic for this
         # 20240807 - TN - set first component as main?
+
+        if (queue is None) and (cstar_sysmgr.scheduler is not None):
```

Comment:
These brackets are redundant in python. I'm suprised the pre-commit linter hasn't removed them automatically. Another example of https://github.com/CWorthy-ocean/C-Star/issues/183

```suggestion
        if queue is None and cstar_sysmgr.scheduler is not None:
```

Reply:
I think we've touched on this before - personal preference, if it's not a problem. Separating logical expressions into little packages like this makes it much more readable to me, especially as they get complex.